pkg/tsdb/cloudwatch/metric_find_query.go

```diff
@@ -196,6 +197,7 @@ func init() {
 		"AWS/TrustedAdvisor":    {},
 		"AWS/VPN":               {"TunnelIpAddress", "VpnId"},
 		"AWS/WorkSpaces":        {"DirectoryId", "WorkspaceId"},
+		"ContainerInsights":     {"ClusterName", "Namespace", "InstanceId", "PodName", "Service"},
```

Comment:
I think that `NodeName` should be added too.

Reply:
Resolved, in next commit